docs/life_of_a_godwoken_transaction.md

```diff
@@ -0,0 +1,338 @@
+    In the above example, the PoASetup includes 2 aggregators, each aggregator gets a round interval of 90 seconds, where it can issue at most one layer 2 block.
+* In an actual transaction unlocking the PoA guarded cell, a different cell named `PoAData` cell must also be updated with PoA information. In the current Godwoken deployment, each transaction contains such `PoAData` cell, an example can be found in output cell #1 of this Godwoken [transaction](https://explorer.nervos.org/aggron/transaction/0x51bee60d0e68bf3735dbaf7ffeea44727cd2112058a22b5a3e259810ce01a0de). The data part for this output cell is:
+
+    ```
+    0xa3860e6000000000a3860e6000000000000000000000
+    ````
+
+    This contains the [PoAData](https://github.com/xxuejie/clerkb/blob/f5455b1bbc3155fdd0e4dd96cf0667cc22098a8b/src/config.ts#L16) data structure in a [customized](https://github.com/xxuejie/clerkb/blob/f5455b1bbc3155fdd0e4dd96cf0667cc22098a8b/src/config.ts#L120) serialization format. The PoA lock script uses PoAData cell to update round internal information, such as starting point of current round interval, and progress in current round interval.
+* The identity of each aggregator is proved using the same [owner lock technique](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0025-simple-udt/0025-simple-udt.md#owner-lock-script) as used in sUDT: the aggregator include a special input cell in the current transaction, the input cell's lock script hash must match the identity included in the PoASetup cell.
+* The aggregators can also start a vote to update the contents in the PoASetup cell. For example, when an aggregator behaves badly, either by not issuing blocks, or issuing too many invalid blocks, the other aggregators can start a vote to upgrade the PoASetup cell, hence removing the misbehaved aggregator. Another example here, is that in the event of congested network, aggregators can vote to alter the round intervals for better performance.
+
+# Actions
+
+In this sections we will explain all the actions one can perform on Godwoken, together with technical details related to each action.
+
+## Deposit
+
+To use Godwoken, one must first deposit some tokens(either CKB or sUDTs) from layer 1 to layer 2. This is named as a **deposit** action. A deposit action is represented as a layer 1 CKB transaction, it must create a special output cell named a **deposition cell**. [Here](https://explorer.nervos.org/aggron/transaction/0x09334d0ab2bf84714f0e51715bb7251a4887aa3066ef5ad0770bd463297c1194) is an example of a deposit action. This transaction deposits 10000 CKB to Godwoken. What is relavant here, is the lock script of output cell #0:
```

Comment:
`deposit` can be used as either noun. or verb., `deposition` is a different word. I think the output should be called `deposit cell`.

Example:
```suggestion
To use Godwoken, one must first deposit some tokens(either CKB or sUDTs) from layer 1 to layer 2. This is named as a **deposit** action. A deposit action is represented as a layer 1 CKB transaction, it must create a special output cell named a **deposit cell**. [Here](https://explorer.nervos.org/aggron/transaction/0x09334d0ab2bf84714f0e51715bb7251a4887aa3066ef5ad0770bd463297c1194) is an example of a deposit action. This transaction deposits 10000 CKB to Godwoken. What is relavant here, is the lock script of output cell #0:
```

There are more 'deposition' occurences in the remaining article.